server/remote/schedule.go

```diff
@@ -29,7 +38,16 @@ type ScheduleInformation struct {
 
 	Error *ScheduleInformationError `json:"error"`
 
-	// ScheduleItems []interface{} `json:"scheduleItems,omitempty"`
+	ScheduleItems []*ScheduleItem `json:"scheduleItems,omitempty"`
 	// WorkingHours interface{} `json:"workingHours,omitempty"`
 	// Error *FreeBusyError `json:"error,omitempty"`
 }
+
+type ScheduleItem struct {
+	IsPrivate bool
+	Status    string
+	Subject   string
+	Location  string
+	Start     DateTime
```

Comment:
To follow suit with `Event` use a pointer here, though I mistakenly suggested to not use a pointer earlier.

```suggestion
	Start     *DateTime
```